frontend/plan/components/selector/CourseInfo.tsx

```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import PropTypes from "prop-types";
 import styled from "styled-components";
 import CourseDetails from "./CourseDetails";
 import SectionList from "./SectionList";
 import { Course } from "../../types";
 
 interface CourseInfoProps {
     course: Course;
-    back: () => void;
+    back?: () => void;
```

Comment:
just for my own clarification -- does this mean back is option? what's the reason it's optional?

Reply:
Yep! This means that the `back` property is either `() => void` or `undefined`, which is basically missing. If there's only one course in the search results, we automatically open its course details and don't provide a back button. The CourseInfo component then isn't passed a function to go back, and so won't show a back button.